samples/csharp/MicrosoftEduGraphSamples/Workflows/ModuleWorkflow.cs

```diff
@@ -0,0 +1,153 @@
+                {
+                    throw new Exception("Missing password please check appconfig.json file.");
+                }
+            }
+            catch (Exception ex)
+            {
+                throw;
+            }
+        }
+        /// <summary>
+        /// Workflow to create and publish the module
+        /// </summary>
+        public async Task ClassworkAsync()
+        {
+            try
+            {
+                // Get a Graph client using delegated permissions
+                var graphClient = GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], _config["teacherAccount"], _config["password"]);
+
+                // Create a draft module.
+                var module = await MicrosoftGraphSDK.Module.CreateAsync(graphClient, _config["classId"], "Sample Module "+ DateTime.Now.ToString("dd/MM/yyyy HHmm"), "This is the first module created using Graph SDK.");
+                Console.WriteLine($"New module has been created: {module.Id} - {module.DisplayName} - {module.Status}");
+                await MicrosoftGraphSDK.Module.SetupResourcesFolder(graphClient, _config["classId"], module.Id);
+
+                // Add a link resource.
+                EducationModuleResource requestBody = new EducationModuleResource
+                {
+                    Resource = new EducationLinkResource
+                    {
+                        OdataType = "#microsoft.graph.educationLinkResource",
+                        DisplayName = "Bing site",
+                        Link = "https://www.bing.com",
+                    },
+                };
+                // Adding a link resource
+                var newResource = await MicrosoftGraphSDK.Module.PostResourceAsync(graphClient, _config["classId"], module.Id.ToString(), requestBody);
+
+                // Add a new word document resource.
+                requestBody = new EducationModuleResource
+                {
+                    Resource = new EducationWordResource
+                    {
+                        OdataType = "#microsoft.graph.educationWordResource",
+                        DisplayName = "test_word_file.docx",
+
+                    },
+                };
+                // A new Word file is added
+                newResource = await MicrosoftGraphSDK.Module.PostResourceAsync(graphClient, _config["classId"], module.Id.ToString(), requestBody);
+
+                // Get the General channel
+                var channels = await MicrosoftGraphSDK.Team.GetChannelsAsync(graphClient, _config["classId"]);
+
+                // Add a channel resource
+                requestBody = new EducationModuleResource
+                {
+                    Resource = new EducationChannelResource
+                    {
+                        OdataType = "#microsoft.graph.educationChannelResource",
+                        Url = "https://graph.microsoft.com/v1.0/teams/"+ _config["classId"] + "/channels/"+ channels.Value[0].Id,
```

Comment:
nit: leave spaces before and after each **+**

Reply:
spaces added